src/js/components/FormField/FormField.js

```diff
@@ -336,11 +345,26 @@ const FormField = forwardRef(
       isFileInputComponent = true;
     }
 
+    let childName;
+    Children.forEach(children, (child) => {
+      if (child && child.type) {
+        childName = child.type.displayName;
+        if (childName?.length > 1)
```

Comment:
```suggestion
        // camelcase component name to match theme object key
        if (childName?.length > 1)
```

Reply:
Why do we check > 1?

Reply:
☝️ also was wondering the same thing. I guess Im assuming to ensure that only components with valid names but if we document this as working with only our inputs then Im not sure wed need the check?

Reply:
Switched this to > 0 instead of > 1

Reply:
Just wanted to check that the string wasn't `''` or undefined before trying to do `childName.charAt(0).toLowerCase() + childName.slice(1)`